src/components/HitherJobMonitor/HitherJobMonitor.tsx

```diff
@@ -1,28 +1,38 @@
 import { Button, IconButton, Link as LinkMui, Table, TableBody, TableCell, TableHead, TableRow } from '@material-ui/core';
 import { Delete } from '@material-ui/icons';
-import React, { Dispatch, FunctionComponent, useState } from 'react';
-import { connect, MapDispatchToProps, MapStateToProps } from 'react-redux';
-import NiceTable from '../components/NiceTable';
-import { HitherJob } from '../extensions/common/hither';
-import { RootAction, RootState } from '../reducers';
+import React, { FunctionComponent, useContext, useEffect, useState } from 'react';
+import { HitherContext, HitherJob } from '../../extensions/common/hither';
+import NiceTable from '../NiceTable';
 
-interface StateProps {
-    allJobs: HitherJob[],
-    pendingJobs: HitherJob[],
-    runningJobs: HitherJob[],
-    finishedJobs: HitherJob[],
-    erroredJobs: HitherJob[]
-}
-
-interface DispatchProps {
-}
+type Props = {}
 
-interface OwnProps {
-}
+const HitherJobMonitor: FunctionComponent<Props> = () => {
+    const [hitherJobs, setHitherJobs] = useState<HitherJob[]>([])
+    const hither = useContext(HitherContext)
+    useEffect(() => {
+        // this should only get called once
+        // (hither should not change, but if it does we might have a problem here)
+        const update = () => {
+            const hj = hither.getHitherJobs()
+            setHitherJobs(hj)
```

Comment:
We do polling to get the updated hither jobs (which is sufficient for monitory changes). This way we don't have an expensive state changes whenever a job status changes. This useEffect with cleanup is a nice mechanism for doing polling of this nature.